ouroboros-consensus/changelog.d/20230306_122628_javier.sagredo_move_hfc_utils.md

```diff
@@ -0,0 +1,24 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+
+### Non-Breaking
+
+- Move `Util` modules that are related only to SOP to `Data.SOP`.
```

Comment:
Maybe mention which modules are going to be deprecated in the future?

Reply:
Done